### PR TITLE
Add a website link on README

### PR DESCRIPTION
**Pull request by wu-sheng**
Advice by @ptone on https://github.com/opentracing/opentracing.io/issues/176. 

I think it's better to have it. @bensigelman

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,5 +1,7 @@
 # The OpenTracing Specification repository
 
+OpenTracing Specification Official Website: [http://opentracing.io/documentation/pages/spec](http://opentracing.io/documentation/pages/spec)
+
```
Comment (yurishkuro) [inline README.md]: missing a period at the end.

Comment (yurishkuro) [inline README.md]: Also, this URL may be better: http://opentracing.io/spec

**Comment by wu-sheng** [comment]
@yurishkuro Changed.

**Comment by yurishkuro** [comment]
Thank, @wu-sheng

**Comment by yurishkuro** [comment]
Sorry, I had to re-commit because of the bad commit message that gets added automatically. I changed the settings on the repo to always squash commits from now on.

**Comment by wu-sheng** [comment]
: )  no need.  Do anything you need to be done.

File: README.md [hunk 2]
```diff
@@ -1,5 +1,7 @@
 # The OpenTracing Specification repository
 
+OpenTracing Specification Official Website: [http://opentracing.io/spec](http://opentracing.io/spec).
```
Comment (bensigelman) [inline README.md]: @wu-sheng this is not the endgame I'd like here... let's close this and I will open an issue to resolve this properly from the other side. https://github.com/opentracing/opentracing.io/issues/177

**Comment by bensigelman** [comment]
@yurishkuro this isn't really the solution I'd like, but I can just clean things up later.

**Comment by wu-sheng** [comment]
What's the better solution?  Maybe I am not total understood.

And Happy holidays . @bensigelman @yurishkuro

**Pull request merged**